common/test/acceptance/fixtures/course.py

```diff
@@ -152,6 +152,18 @@ def __str__(self):
         """
         return "<CourseFixture: org='{org}', number='{number}', run='{run}'>".format(**self._course_dict)
 
+    def update_course_details(self, course_details):
+        """
+        update course settings (e.g. start and end date)
```

Comment:
Can we make doc string of this setter consistent with existing setter methods. some thing like "Adds course details to be set on the course when the install method is called."